ingestion/src/metadata/ingestion/lineage/sql_lineage.py

```diff
@@ -163,6 +171,116 @@ def get_table_fqn_from_query_name(
     return database_query, schema_query, table
 
 
+def handle_udf_column_lineage(
+    column_lineage_original: dict,
+    column_lineage_generated: List[Tuple[Column, Column]],
+):
```

Comment:
I think you could also extract each loop into its own method like this

```
for k0, v0 in d.items():
    print("k0: ", k0)
    print("v0: ", dict(v0))
    for k1, v1 in v0.items():
        print("k1: ", k1)
        print("v1: ", dict(v1))
        for k2, v2 in v1.items():
            print("k2: ", k2)
            print("v2: ", dict(v2))
            for k3, v3 in v2.items():
                print("k3: ", k3)
                print("v3: ", dict(v3))
                for k4, v4 in v3.items():
                    print("k4: ", k4)
                    print("v4: ", v4)
```
becomes
```
def processV1(k, v):
    for k, v in v.items():
        print("k1: ", k)
        print("v1: ",dict( v))
        processV2(k, v)

def processV2(k, v):
    for k, v in v.items():
        print("k2: ", k)
        print("v2: ",dict( v))
        processV3(k, v)

def processV3(k, v):
    for k, v in v.items():
        print("k3: ", k)
        print("v3: ",dict( v))
        processV4(k, v)

def processV4(k, v):
    for k, v in v.items():
        print("k4: ", v)
        print("v4: ",v)


for k0, v0 in d.items():
    print("k0: ", k0)
    print("v0: ", dict(v0))
    processV1(k0, v0)
```